oauth2_http/javatests/com/google/auth/http/HttpCredentialsAdapterTest.java

```diff
@@ -45,4 +57,77 @@ public void initialize_populatesOAuth2Credentials() throws IOException {
     String authorizationHeader = requestHeaders.getAuthorization();
     assertEquals(authorizationHeader, expectedAuthorization);
   }
+
+  public MockLowLevelHttpResponse makeErrorResponse() {
```

Comment:
I would structure this slightly differently. Public helpers break up the readability of tests a bit. I'd suggest adding these to MockTokenServerTransport.